azure-pipelines.yml

```diff
@@ -42,13 +42,23 @@ jobs:
   pool:
     vmImage: 'ubuntu-18.04'
   steps:
+  - bash: |
```

Comment:
Doesn't the Go Installer task work? https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/go-tool?view=azure-devops